src/components/balance/Accounts.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="tw-flex tw-flex-col md:tw-flex-row tw-gap-y-8 tw-gap-x-8 tw-flex-wrap">
+    <template v-if="isMetamask">
```

Comment:
I don't think this method of displaying is scalable. For example, what if we want to add foreign currencies like XCMP and bridge tokens?
I suggest rendering this from a array of accounts.